itenium-socks/src/app/socks/sock-reviews.component.html

```diff
@@ -9,13 +9,16 @@ <h2>
   <div class="container px-0">
     <div id="customCarousel2" class="carousel carousel-fade" data-ride="carousel">
       <div class="carousel-inner">
-        <div *ngFor="let review of reviews$ | async; index as i" class="carousel-item {{ i == 0 ? 'active' : '' }}">
+        @for (review of reviews$ | async; track review) {
+        <div class="carousel-item {{ $index == 0 ? 'active' : '' }}">
           <div class="box">
             <div class="client_info">
               <div class="client_name">
                 <h5>
                   {{ review.socksId }}
-                  <i *ngFor="let _ of [].constructor(review.rating)" class="fa fa-star" style="color: gold"></i>
+                  @for (rating of [].constructor(review.rating); track rating) {
```

Comment:
De `track` is om Angular duidelijk te maken wat het verschil is tussen de verschillende "values" om zo te zien wanneer twee values dezelfde zijn
In dit geval is de "rating" elke keer empty, niet ideaal

We zouden hier een array kunnen maken ala [1, 2, 3, 4, 5] en de track houden.

Op zich is het hier niet zo belangrijk omdat er toch geen wijzigingen zijn aan de rating